chore(ui5-multi-combobox): migrate tests to cypress

nnaydenow opened the pull request.

Vonahz commented on packages/main/cypress/specs/MultiComboBox.cy.tsx:
```diff
@@ -3905,4 +3905,212 @@ describe("Keyboard Handling", () => {
 	});
+
+	describe("Copy/Cut/Paste keyboard shortcuts", () => {
+		const dispatchCopyEvent = () => {
```
This name is wrong? It actually dispatches a pasteEvent.

nnaydenow commented:
Hi @ndeshev, 

Please take a look as I removed the spec left in https://github.com/UI5/webcomponents/pull/12289.

-----

Hi @ilhan007,

Please take a look as I removed the test suite scripts for main package. In follow up change I'll remove it from github workflows, package.json and etc..
